### PR TITLE
Add  support for configurable OTLP export strategy with cumulative default strategy

### PR DESCRIPTION
**Pull request by huyan0**
This PR describes and records discussions around spec [#731](https://github.com/open-telemetry/opentelemetry-specification/issues/731), [#725](https://github.com/open-telemetry/opentelemetry-specification/issues/725) and the overall conclusion from Metrics SIG meetings. 

cc: @alolita @huyan0

### CODE REVIEW

**Review by connorlindsey — APPROVED** [review]

**Review by CunjunWang — COMMENTED** [review]
LGTM

**Review by connorlindsey — APPROVED** [review]

File: text/metrics/0000-otlp-export-behavior.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Related issues: spec [#731](https://github.com/open-telemetry/opentelemetry-specification/issues/731), [#725](https://github.com/open-telemetry/opentelemetry-specification/issues/725)
+
+Sig meeting discussion: [Metrics SIG](https://docs.google.com/document/d/1LfDVyBJlIewwm3a0JtDtEjkusZjzQE3IAix8b0Fxy3Y/edit#heading=h.fxqkpi2ya3br) July 23, July 30
+
+## Explanation
+
+To support Prometheus as well as delta backends, the SDK should be configurable and supports OTLP exporter to have both cumulative default or delta export behavior. The implication is that the OTLP metric protocol should support both cumulative and delta report strategy.
+
+Users should be allowed to declare an environment variable or configuration field that determines this setting for OTLP exporters.
```
Comment (jmacd) [inline text/metrics/0000-otlp-export-behavior.md]: Maybe suggest a value?

**Review by jmacd — APPROVED** [review]
This is :100: great!

File: text/metrics/0000-otlp-export-behavior.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## Internal details
+OTLP exporters can report the strategy it desires to the Metric SDK, and the SDK can merge the previous state of metrics with current value and return the appropriate values to the exporter.  
+  
+The machinery for configurable export behavior is already demonstrated by the Metrics Processor component in [Go SDK](https://github.com/open-telemetry/opentelemetry-go/pull/840). Also, the current OTLP metrics definition in [PR #193](https://github.com/open-telemetry/opentelemetry-proto/pull/193) can support both strategies. 
+
+## Trade-offs and mitigations
+
+ To support cumulative export strategy, the SDK needs to maintain state of each cumulative metrics. This means users with high-cardinality metrics can experience high memory usage. 
+ 
+This problem could be addressed by adding support in the Collector, when configured as an Agent, to support converting delat OTLP to Cumulative OTLP, but this requires a single agent for each metric generating client so that all delta values of a metric are converted at the same location.
```
Comment (huyan0) [inline text/metrics/0000-otlp-export-behavior.md]: typo: delta

File: text/metrics/0000-otlp-export-behavior.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# OTLP Configurable Export Behavior
+ Add OTLP exporters support for configurable export behavior.
+
+ The export behaviors are exporting cumulative values since start time by default, and exporting delta values per collection interval when configured. 
+
+## Motivation 
+
+1. Metric backends such as Prometheus and Prometheus Remote Write integrated storages requires cumulative values for cumulative/adding metrics rather than delta value per collection interval. Thus, to export metrics generated by the SDK using the collector, incoming values from the SDK should be in cumulative form. However, backends like Statsd expects delta value of each collection interval. To support different backends, OTLP metric export behavior should be configurable, with cumulative export as default. The metrics SIG meeting concluded that cumulative export behavior is more reliable, as explained in the following section.
```
Comment (dabangarang) [inline text/metrics/0000-otlp-export-behavior.md]: Maybe say "backends that make use of the Prometheus Remote Write API"

Comment (alolita) [reply text/metrics/0000-otlp-export-behavior.md]: # OTLP Configurable Export Behavior
 Add support for configurable behavior in OTLP exporters

**Pull request merged**